src/Handler/TagHandler.php

```diff
@@ -57,19 +59,20 @@ public function tagResponse(Response $response, $replace = false)
         $tags = [];
         if (!$replace && $response->headers->has($this->tagsHeader)) {
             $headers = $response->headers->get($this->tagsHeader, null, false);
-            if (!empty($headers)) {
+            if ($headers) {
                 // handle both both comma (FOS) and space (this bundle/xkey/fastly) separated strings
                 // As there can be more requests going on, we don't add these to tag handler (ez-user-context-hash)
                 $tags = preg_split("/[\s,]+/", implode(' ', $headers));
             }
         }
 
         if ($this->hasTags()) {
+            // note that 'getTagsHeaderValue' returns unique tags
```

Comment:
yes, but after merge they are not guaranteed to be unique right?

Reply:
right, missed that, fixed